pkgs/applications/office/paperless-ngx/default.nix

```diff
@@ -46,14 +54,26 @@ let
     pname = "paperless-ngx-frontend";
     inherit version src;
 
-    npmDepsHash = "sha256-uDaZ7j7IDgKy7wCWND2xzR1qHwUtdyjR4eyIAVy01dM=";
+    npmDepsHash = "sha256-K7wTYGGwEhPoXdRD+4swhSlMH0iem6YkF0tjnVHh7K8=";
 
     nativeBuildInputs = [
       python3
     ] ++ lib.optionals stdenv.isDarwin [
       xcbuild
     ];
 
+    buildInputs = [
+      cairo
+      giflib
+      libjpeg
+      libpng
```

Comment:
According to https://github.com/Automattic/node-canvas#compiling we only need this on darwin/bsd.

Reply:
Confirmed that I was able to get rid of most of these unnecessary packages. However, we *do* need `pango` as a buildInput in order to get the package to build. Putting it in nativeBuildInputs didn't work for me.